pkg/skaffold/docker/dependencies.go

```diff
@@ -42,30 +54,48 @@ func NormalizeDockerfilePath(context, dockerfile string) (string, error) {
 	return filepath.Abs(rel)
 }
 
-// GetDependencies finds the sources dependencies for the given docker artifact.
+// GetDependencies finds the sources dependency for the given docker artifact.
 // All paths are relative to the workspace.
 func GetDependencies(ctx context.Context, workspace string, dockerfilePath string, buildArgs map[string]*string, cfg Config) ([]string, error) {
```

Comment:
Since this is doing exactly what [SyncStore](https://github.com/GoogleContainerTools/skaffold/blob/6c9376c20168314d838162c8350f45bc2e802c60/pkg/skaffold/util/store.go#L24) provides should we use that instead of add new package dependencies?

Reply:
Or modify the `SyncStore` implementation to this since this looks to be using fewer underlying constructs.